### PR TITLE
Abstract and Overview cleanups from a real tech writer

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
Thanks @carolhoward!

File: spec/overview.md [hunk 1]
```diff
@@ -40,14 +40,15 @@ DID. For resolvers that want more assurance, `did:tdw` provides a way to "trust
 did:web" (or to enable a "trusted web" if you say it fast) enabled by the
 features listed in the [Abstract](#abstract).
 
-The following is a `tl;dr` summary of how `did:tdw` works.
+The following is a `tl;dr` summary of how `did:tdw` works:
 
-1. In the same path as where DID resolvers find a `did:web`'s `did.json`,
-  `did:tdw`'s `did.jsonl` ([[ref: JSON Lines]]) file is found. The same `did:web` DID-to-HTTPS
-  transformation is used.
+1. `did.tdw` uses the same DID-to-HTTPS tranformation as `did:web`, so
```
Comment (brianorwhatever) [inline spec/overview.md:45]:
```suggestion
1. `did:tdw` uses the same DID-to-HTTPS tranformation as `did:web`, so
```

**Review by brianorwhatever — APPROVED** [review]
Very good thanks @carolhoward

File: spec/overview.md (hunk 1 above)
Comment (swcurran) [reply spec/overview.md]: Fixed

**Review by brianorwhatever — APPROVED** [review]

**Pull request merged**